packages/web-twig/src/Resources/components/Pagination/Pagination.stories.twig

```diff
@@ -0,0 +1,29 @@
+{% extends 'layout/plain.html.twig' %}
+
+{% block content %}
+
+{# Pagination Current First #}
+<section class="docs-Section">
+    <h2 class="docs-Heading">Pagination Current First</h2>
+    {% include '@components/Pagination/stories/PaginationCurrentFirst.twig' %}
+</section>
```

Comment:
Thanks for the headings! Just one more thing: we always have the section with heading inside the included story… Would you fix that please? 🙏🏻

Reply:
That is not true. We want to use the stories in the future for rendering in Supernova or for snapshots tests. In that case, the heading will be a problem, I think.